### PR TITLE
fix: 10558 remove AbstractEnumConfigConverter

### PR DESCRIPTION
**Pull request by mxtartaglia-sl**
**Description**:
Removes the uses of AbstractEnumConfigConverter and its subclasses. Once 10535 is available, this becomes an out of the box functionality for the ConvertionService.

**Related issue(s)**:
Requieres: https://github.com/hashgraph/hedera-services/issues/10535
Fixes (https://github.com/hashgraph/hedera-services/issues/10558)

### CODE REVIEW

**Review by hendrikebbers — APPROVED** [review]

**Review by cody-littley — APPROVED** [review]
Approved for files owned by hashgraph team.

**Review by alittley — COMMENTED** [review]
Approve wrt file owned by hashgraph

**Review by alittley — APPROVED** [review]

**Review by imalygin — APPROVED** [review]

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/config/BootstrapConfigProviderImpl.java [hunk 1]
```diff
@@ -60,7 +59,6 @@ public BootstrapConfigProviderImpl() {
                 .withConfigDataType(LedgerConfig.class)
                 .withConverter(new BytesConverter())
                 .withConverter(new SemanticVersionConverter())
-                .withConverter(new ProfileConverter())
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/config/BootstrapConfigProviderImpl.java:63]: So these are automatically converted now  without converter explicitly as I understand. Is that correct ?

**Review by Neeharika-Sompalli — APPROVED** [review]
Makes sense to me. Thanks @mxtartaglia-sl

**Review by Neeharika-Sompalli — APPROVED** [review]
Makes sense to me. Thanks @mxtartaglia-sl

**Review by JeffreyDallas — APPROVED** [review]

**Pull request merged**